bellows/cli/dump.py

```diff
@@ -60,6 +77,12 @@ async def _dump(ctx, channel, outfile):
     def cb(frame_name, response):
         if frame_name == "mfglibRxHandler":
             data = response[2]
+            fcs = ieee_15_4_fcs(data[0:-2])
+
+            if data[-2:] != fcs:
```

Comment:
I would not do it unconditionally, but only if ezsp version == 8 and last two bytes of the frame that static value